articles/cms/wordpress/configuration.md

```diff
@@ -62,19 +61,50 @@ lock.once('signin ready', function() {
     var iconList = $(this.$container).find('.a0-iconlist');
     iconList.append(link);
 });
-```
+  ```
 
-  **Note:** The variable `lock` refers to an instance of the Login Widget.
-* **Auth0 Implicit Flow:** If enabled, uses the [Implicit Flow](/protocols#oauth-for-native-clients-and-javascript-in-the-browser) protocol for authorization in cases where the server is without internet access or behind a firewall. If this setting is enabled with **SSO**, you must add `http://your-domain/wp-login.php` as a valid callback in your Auth0 app.
+  **NOTE:** The variable `lock` refers to an instance of the Login Widget.
+* **Username style:** Selecting **Email** will require users to enter their email address to login. Set this to *username* if you do not want to force a username to be a valid email address.
+* **Remember last login:** Requests SSO data and enables the *Last time you signed in with[...]* option. For more information,  see [rememberLastLogin {Boolean}](/libraries/lock/customization#rememberlastlogin-boolean-).
+* **Translation:** A valid JSON object representing the Lock's dict parameter. The 'dict' parameter can be a string matching any supported language ('en', 'es', 'it', etc...) or an object containing customized label text. If set, this will override the Title setting. For more info see [dict {String|Object}](/libraries/lock/customization#dict-string-object-).
+
+### Connections
+
+Enable the supported social identity providers you want to allow users to login with. You can configure your own app keys and settings for these connections in the [Auth0 Dashboard](${uiURL}/#/connections/social).
+
+### Advanced
+
+* **Use passwordless login:** Enable this option to replace the login widget with Lock Passwordless.
+* **Widget URL:** The URL of to the latest available widget in the CDN.
+* **Connections:** List here each of the connections the Lock widget will allow users to select from. If blank, any connection choice will be allowed. If passwordless login is enabled, you must list all allowable connections here.
+* **Remember users session:** By default, user sessions live for two days. Enable this setting to keep user sessions live for 14 days.
+* **Link users with same email:** This option enables the linking of accounts with the same verified e-mail address.
+* **Twitter consumer key:**
```

Comment:
Where do they get the twitter and facebook credentials? I will add those instructions here.